shell/app-shell/app-shell.js

```diff
@@ -160,6 +160,12 @@ class AppShell extends Xen.Base {
       profileSoloPath: '../web/artifacts/profile.manifest'
     };
   }
+  _setState(state) {
+    if (super._setState(state)) {
+      log(state);
```

Comment:
Still needed?

Reply:
(I see this same snippet in `arc-cloud.js` and other files too.)

Reply:
I believe this is useful logging. I may incorporate into the base class, haven't decided yet.

Reply:
👍